docker-run

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+run_docker () {
+    docker run -it --rm -v $(pwd)/generated_docs:/work/streisand/generated_docs \
+	   -v $HOME/.ssh:/work/streisand/sshkeys \
+	   -e STREISAND_SSH_KEY_FROM_HOST="/work/streisand/sshkeys/id_rsa.pub" \
+	   -e STREISAND_SSH_ADD_KEY=1 \
+	   streisand-builder ssh-agent "$@"
+}
+
+if [ -z "$1" ]
+then
+    run_docker /work/streisand/streisand
```

Comment:
Is this redundant with the `CMD` in the Dockerfile? Could you do `run_docker` here instead of `run_docker /work/streisand/streisand` and let the `CMD` specify the default path?

Reply:
Yes, and I've eliminated that cruft.